INSTALL.rst

```diff
@@ -152,6 +257,15 @@ You can now load the demo records (licenses, funders, grants, records):
    (zenodo)$ zenodo migration reindex -t recid
    (zenodo)$ zenodo index run -d
 
+Finally, run the Zenodo application:
+
+.. code-block:: console
+
+    (zenodo)$ export FLASK_DEBUG=True
+    (zenodo)$ zenodo run
```

Comment:
I think FLASK_DEBUG is already live reloading, but I guess it's a good idea to document those :+1: